test/image/mocks/colorscale_opacity.json

```diff
@@ -1,6 +1,523 @@
 {
     "data": [
         {
+         "type": "contour",
```

Comment:
might want to change the title to match what's actually being tested in this image (scatter & contour colorscale opacity)